envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/LocalReplyMappingTest.kt

```diff
@@ -0,0 +1,113 @@
+package pl.allegro.tech.servicemesh.envoycontrol
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.RegisterExtension
+import pl.allegro.tech.servicemesh.envoycontrol.assertions.isUnreachable
+import pl.allegro.tech.servicemesh.envoycontrol.assertions.untilAsserted
+import pl.allegro.tech.servicemesh.envoycontrol.config.consul.ConsulExtension
+import pl.allegro.tech.servicemesh.envoycontrol.config.envoy.EnvoyExtension
+import pl.allegro.tech.servicemesh.envoycontrol.config.envoycontrol.EnvoyControlExtension
+import pl.allegro.tech.servicemesh.envoycontrol.config.service.EchoServiceExtension
+
+class LocalReplyMappingTest {
+
+    companion object {
+
+        @JvmField
+        @RegisterExtension
+        val consul = ConsulExtension()
+
+        @JvmField
+        @RegisterExtension
+        val envoyControl = EnvoyControlExtension(
+            consul, mapOf(
+                "envoy-control.envoy.snapshot.routing.service-tags.enabled" to true,
+                "envoy-control.envoy.snapshot.routing.service-tags.metadata-key" to "tag",
+                "envoy-control.envoy.snapshot.dynamic-listeners.local-reply-mapper.enabled" to true,
```

Comment:
You can save `envoy-control.envoy.snapshot.dynamic-listeners.local-reply-mapper.` to a prefix it will be easier to read.